plugins/gimbal/gimbal_impl.cpp

```diff
@@ -49,6 +49,28 @@ void GimbalImpl::set_pitch_and_yaw_async(float pitch_deg, float yaw_deg,
         MavlinkCommands::DEFAULT_COMPONENT_ID_AUTOPILOT);
 }
 
+Gimbal::Result GimbalImpl::set_roi_location(double latitude_deg, double longitude_deg,
+                                            float altitude_m)
+{
+    return gimbal_result_from_command_result(
+               _parent.send_command_with_ack(
```

Comment:
@julianoes it would be better to use `COMMAND_INT` here, but I didn't see it being available.

Reply:
Right! I made an issue: https://github.com/dronecore/DroneCore/issues/326.

Reply:
Thanks!